docs/build/html/python_pachyderm.experimental.proto.v2.admin_v2.html

```diff
@@ -117,7 +117,7 @@ <h1>python_pachyderm.experimental.proto.v2.admin_v2 package<a class="headerlink"
 
 <dl class="py class">
 <dt class="sig sig-object py" id="python_pachyderm.experimental.proto.v2.admin_v2.ClusterInfo">
-<em class="property"><span class="pre">class</span><span class="w"> </span></em><span class="sig-name descname"><span class="pre">ClusterInfo</span></span><span class="sig-paren">(</span><em class="sig-param"><span class="n"><span class="pre">id:</span> <span class="pre">str</span> <span class="pre">=</span> <span class="pre">&lt;object</span> <span class="pre">object</span> <span class="pre">at</span> <span class="pre">0x107efd2f0&gt;</span></span></em>, <em class="sig-param"><span class="n"><span class="pre">deployment_id:</span> <span class="pre">str</span> <span class="pre">=</span> <span class="pre">&lt;object</span> <span class="pre">object</span> <span class="pre">at</span> <span class="pre">0x107efd2f0&gt;</span></span></em><span class="sig-paren">)</span><a class="reference internal" href="_modules/python_pachyderm/experimental/proto/v2/admin_v2.html#ClusterInfo"><span class="viewcode-link"><span class="pre">[source]</span></span></a><a class="headerlink" href="#python_pachyderm.experimental.proto.v2.admin_v2.ClusterInfo" title="Permalink to this definition"></a></dt>
+<em class="property"><span class="pre">class</span><span class="w"> </span></em><span class="sig-name descname"><span class="pre">ClusterInfo</span></span><span class="sig-paren">(</span><em class="sig-param"><span class="n"><span class="pre">id:</span> <span class="pre">str</span> <span class="pre">=</span> <span class="pre">&lt;object</span> <span class="pre">object</span> <span class="pre">at</span> <span class="pre">0x104440160&gt;</span></span></em>, <em class="sig-param"><span class="n"><span class="pre">deployment_id:</span> <span class="pre">str</span> <span class="pre">=</span> <span class="pre">&lt;object</span> <span class="pre">object</span> <span class="pre">at</span> <span class="pre">0x104440160&gt;</span></span></em><span class="sig-paren">)</span><a class="reference internal" href="_modules/python_pachyderm/experimental/proto/v2/admin_v2.html#ClusterInfo"><span class="viewcode-link"><span class="pre">[source]</span></span></a><a class="headerlink" href="#python_pachyderm.experimental.proto.v2.admin_v2.ClusterInfo" title="Permalink to this definition"></a></dt>
```

Comment:
This change is made a lot can you briefly explain what it is?

Reply:
These appears to be documentation for the return type which is one of the gRPC generated objects. I'll need to do some investigating as to why it's unable to properly document this, but it might be worth getting at least this documentation out and then fixing this.